pkg/cluster/internal/kubeadm/config.go

```diff
@@ -445,12 +481,19 @@ evictionHard:
   nodefs.available: "0%"
   nodefs.inodesFree: "0%"
   imagefs.available: "0%"
+{{if .FeatureGates}}featureGates:
+{{ range $key, $value := .FeatureGates }}
```

Comment:
not sure if this needs to be deterministically ordered, but if it's a map it won't be